### ISSUE TITLE
Couldn't find the function prepare_node_explanation_task in the eval_utils.py file

### ISSUE THREAD

**Issue by DayaAlex** [agapiR/rdl-explain#1]
The function prepare_node_explanation_task  is used in learn_masks.py, learn_filter_masks.py and eval_utils.py. So I'm not able to test your code as the explainer is using the explanation_task returned by that function.

    explanation_task = prepare_node_explanation_task(task, predictions, explanation_target_type=explanation_target_type)
    del task # Delete original task to free memory

    # Initialize the explainer
    explainer = RDLExplainer(config, model_to_explain, data, explanation_task)

    # Learn explanation masks
    mask, mask_vals, metrics = explainer.learn_masks(
                                            eps = reg_eps,
                                            explanation_type = explanation_type, 
                                            elimination_strategy = elimination_strategy,
                                            n_epochs = epochs,
                                            lr = learning_rate,
                                        )
